qa/rpc-tests/sc_create.py

```diff
@@ -44,29 +43,6 @@ def setup_network(self, split=False):
         self.is_network_split = split
         self.sync_all()
 
-    def disconnect_nodes(self, from_connection, node_num):
-        ip_port = "127.0.0.1:" + str(p2p_port(node_num))
-        from_connection.disconnectnode(ip_port)
-        # poll until version handshake complete to avoid race conditions
-        # with transaction relaying
-        while any(peer['version'] == 0 for peer in from_connection.getpeerinfo()):
-            time.sleep(0.1)
-
-    def split_network(self):
-        # Split the network of three nodes into nodes 0-1 and 2.
-        assert not self.is_network_split
-        self.disconnect_nodes(self.nodes[1], 2)
-        self.disconnect_nodes(self.nodes[2], 1)
-        self.is_network_split = True
-
-    def join_network(self):
-        # Join the (previously split) network pieces together: 0-1-2
-        assert self.is_network_split
-        connect_nodes_bi(self.nodes, 1, 2)
-        connect_nodes_bi(self.nodes, 2, 1)
-        time.sleep(2)
-        self.is_network_split = False
-
     def dump_sc_info_record(self, info, i):
```

Comment:
util routines might be moved in the py framework, since they are redefined similarly also by other tests